# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/protocol/http/nsHttpChannel.cpp
# Commit: f2df73fda468
# Full Hash: f2df73fda46859150605f33bcdaeb49ab7037720
# Author: Jean-Yves Avenard <jyavenard@mozilla.com>
# Date: 2019-10-19 21:40:34
# Regressor Bug: 1588899
# File Overlap Count: 1
# Description:
#   Bug 1588899 - P1. Move classification flags related method to nsIClassifiedChannel. r=Ehsan,baku
#   
#   This is where it should have been in the first place. Those attributes belong there.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D49577
# ==============================================================================

diff -r b98968eb71d3 -r f2df73fda468 netwerk/protocol/http/nsHttpChannel.cpp
--- a/netwerk/protocol/http/nsHttpChannel.cpp	Thu Oct 17 15:24:31 2019 +0000
+++ b/netwerk/protocol/http/nsHttpChannel.cpp	Sat Oct 19 04:30:24 2019 +0000
@@ -1335,7 +1335,7 @@
     uint32_t flags = isThirdParty ? mThirdPartyClassificationFlags
                                   : mFirstPartyClassificationFlags;
 
-    using CF = nsIHttpChannel::ClassificationFlags;
+    using CF = nsIClassifiedChannel::ClassificationFlags;
     using TC = HttpTrafficAnalyzer::TrackingClassification;
 
     if (flags & CF::CLASSIFIED_TRACKING_CONTENT) {